### PR TITLE
Apply templated resources

### PR DESCRIPTION
**Pull request by itdove**
Signed-off-by: Dominique Vernier <dvernier@redhat.com>

This plugin allows you to apply templated resources. The templated resources are `golang text/template` files and you can pass a yaml containing the values for the parameters.

https://github.com/stolostron/applier/blob/main/README.md

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

File: plugins/applier.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Copyright Contributors to the Open Cluster Management project
+
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: applier
+spec:
+  version: v1.2.1
+  homepage: https://github.com/stolostron/applier
+  shortDescription: Provides an apply and render commands for template based resources.
```
Comment (ahmetb) [inline plugins/applier.yaml:10]: a better and more concise short description:

> Deploy/render resources written as Go templates

**Comment by ahmetb** [comment]
I think the idea makes sense, it's an OK plugin to have since it actually makes k8s API calls as well. The name is also unique enough that it doesn't warrant additional clarification (although a name indicating templating or go would be more ideal).
I only have a small comment.
/approve

**Comment by ahmetb** [comment]
/lgtm
/approve
Thanks

I hope you can add a bit more docs/examples on your repo. I couldn't visualize how it works. 

Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

**Comment by itdove** [comment]
Thank you, I will add examples as much as I can either on how to use it as a Go package or as a CLI.
PS: doing these improvements during my free time ;-)

**Pull request merged**